ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -708,7 +708,10 @@ static void HandleKernelRequest(void* messageMemory, uint32_t messageSize)
 static PrjFS_Result HandleEnumerateDirectoryRequest(const MessageHeader* request, const char* relativePath)
 {
 #ifdef DEBUG
-    cout << "PrjFSLib.HandleEnumerateDirectoryRequest: " << relativePath << endl;
+    cout << "PrjFSLib.HandleEnumerateDirectoryRequest: "
+    << relativePath
```

Comment:
Style nit: perhaps we could vertically align the `<<` operators here. This is already the case on some of the other log expressions you edited, so it's probably best to follow that style.